### ISSUE TITLE
Missing ex_unit and formatter

### ISSUE THREAD

**Issue by manuel-rubio** [karolsluszniak/ex_check#43]
I'm experiencing an issue with Elixir 1.16.1 (Erlang/OTP 26), in my project (it's not an umbrella project) it's giving me this report:

```
✓ compiler success in 0:03
 ✓ credo success in 0:03
 ✓ dialyzer success in 0:08
 ✓ doctor success in 0:02
 ✓ ex_doc success in 0:04
 ✓ mix_audit success in 0:03
 ✓ unused_deps success in 0:01
   ex_unit skipped due to missing file ../test
   formatter skipped due to missing file ../.formatter.exs
   sobelow skipped due to missing package sobelow
```

why is it trying to get these files from an upper directory `../test` and `../.formatter.exs`?

By the way, `mix format --check-formatted` and `mix test` work perfectly if I run them manually.

**Comment by manuel-rubio** [comment]
As a side note, looks like it's happening only with Elixir 1.16, using Elixir 1.15 works well.

**Comment by badeyeye1** [comment]
I experience somewhat similar issue in umbrella app
```elixir
 ✓ compiler success in 0:02
 ✓ sobelow in backend success in 0:01
 ✓ sobelow in web success in 0:01
 ✓ unused_deps success in 0:01
   credo skipped due to missing package credo
   dialyzer skipped due to missing package dialyxir
   doctor skipped due to missing package doctor
   ex_doc skipped due to missing package ex_doc
   ex_unit in backend skipped due to missing file ../apps/backend/test
   ex_unit in web skipped due to missing file ../apps/web/test
   formatter skipped due to missing file ../.formatter.exs
   mix_audit skipped due to missing package mix_audit
```

**Comment by karolsluszniak** [comment]
This is now fixed thanks to https://github.com/karolsluszniak/ex_check/issues/41 and released in `0.16.0`.
